VOH substraction feature added

vSebas opened the pull request.

vSebas committed:
VOH substraction feature added

ivanacollg commented on usv_control/src/velocity_obstacle.cpp:
```diff
@@ -344,17 +344,57 @@ void reachable_velocities(){
-    
+    speed_th = obs_dist / time_horizon_;
+    obstacle_theta = atan2(obstacle_list_[i].x, obstacle_list_[i].y);
+    vel_th_x = speed_th * sin(obstacle_theta);
```
Usar structs

vSebas replied:
Done

ivanacollg commented on usv_control/src/velocity_obstacle.cpp:
```diff
@@ -344,17 +344,57 @@ void reachable_velocities(){
+    intersect_r_y =  numerator / denominator;
+
+    // Construct VOH
+    VOH.clear();
```
Incluir vertices en Cono C, borrar VOH

vSebas replied:
Done

vSebas committed:
Changes:
Vertex struct used
Added VOH vertex directly to the input cones
Commented some lines
Added a README

vSebas committed:
Used more structs

ivanacollg committed:
Commentary changes
